### PR TITLE
Enable leaderelection

### PR DESCRIPTION
**Pull request by Invictus17**
Linking it to https://github.com/kubernetes-client/python/pull/1218

### CODE REVIEW

**Comment by yliaog** [comment]
the module kubernetes/base in this repo needs to be updated to have 'leaderelection'

File: kubernetes/leaderelection [hunk 1]
```diff
@@ -0,0 +1 @@
+base/leaderelection
```
Comment (roycaihw) [inline kubernetes/leaderelection:1]: looks like this is a text file, which caused the import failure. Instead this should be a symbolic link. See https://github.com/kubernetes-client/python/blob/master/kubernetes/dynamic as an example

**Comment by Invictus17** [comment]
There are failures in the 'kubernetes/e2e_test/test_client.py' test which I think are unrelated to my PR.
https://travis-ci.org/github/kubernetes-client/python/builds/757635573

**Comment by yliaog** [comment]
close and reopen to rerun the ci tests

**Pull request merged**